frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -59,18 +59,42 @@ const CampCreationPage = (): React.ReactElement => {
 
   let isCampDetailsFilled = false;
 
+  const startTimeBeforeEndTime = useMemo(() => {
+    try {
+      const [startHours, startMinutes] = startTime
+        .split(":")
+        .map((num) => parseInt(num, 10));
+      const [endHours, endMinutes] = endTime
+        .split(":")
+        .map((num) => parseInt(num, 10));
+
+      if (!(startHours && startMinutes && endHours && endMinutes)) {
```

Comment:
Replace this if condition with:
```
if (Number.isNaN(startHours) || Number.isNaN(endHours)) {
  return true;
}
```
Right now, if you have times with 0, these will also pass the validation due to falsey value logic. Also, there's no need to check for startMinutes in the if condition, if either hours or minutes are not present, then hours will return NaN.